graphistry/tests/test_arrow_uploader.py

```diff
@@ -156,25 +156,25 @@ def test_au_e_enc_full(self):
         }
 
     def test_cascade_privacy_settings_default_global(self):
-        PyGraphistry._config["privacy"] = None
+        PyGraphistry.session.privacy = None
```

Comment:
as we start pushing the client driver -- not this pr -- probably want more testing of the new setup